app/lib/screens/wallets/send.dart

```diff
@@ -385,6 +409,16 @@ class _WalletSendScreenState extends State<WalletSendScreen> {
     return result.code;
   }
 
+  calculateAmount(int percentage) {
+    final fee = chainType == ChainType.Stellar ? 0.1 : 0.01;
+    final amount = double.parse(chainType == ChainType.TFChain
+                ? widget.wallet.tfchainBalance
+                : widget.wallet.stellarBalance) *
+            (percentage / 100) -

```

Comment:
in case of 0 balance, selecting any of the percentages will lead to a negative balance